api-docs/api-reference/methods/domain-trusts/get-list-domain-trust-roles.rst

```diff
@@ -0,0 +1,84 @@
+.. _get-list-domain-trust-roles:
+
+List domain trust roles
+~~~~~~~~~~~~~~~~~~~~~~~
+
+.. code::
+
+    GET /v2.0/RAX-AUTH/trusts/{trustId}/roles
+
+List domain trust roles.
+
+.. note::
+
+    - A user-admin and user-manager can list a trust role if their domain
+      matches the principal domain or delegate domain on the trust.
+    - Users with the role ``identity:domain-trust-admin`` can list domain
+      trusts roles for any principal and delegate domain.
+    - Use the RAX-AUTH API extension to the Core Identity API to implement this
+      API operation.
+
+The following table shows the possible response codes for this operation:
+
+.. csv-table::
+  :header: Response Code, Name, Description
+  :widths: 2, 2, 2
+
+  200, OK, The request succeeded.
+  400, Bad Request, The request was invalid.
+  401, Unauthorized, "You are not authorized to complete this, operation. This error can occur if the, request is submitted with an invalid, authentication token."
```

Comment:
remove the quotes around the message

Reply:
Quotes are needed here to correctly format the table since the message contains commas.

Reply:
Fair enough!